### PR TITLE
Map the remaining static Text fields

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen.mapping [hunk 1]
```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_415 net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen
 	FIELD field_2438 confirmButton Lnet/minecraft/class_4185;
 	FIELD field_2441 biomeSelectionList Lnet/minecraft/class_415$class_4190;
+	FIELD field_26535 BUFFET_BIOME_TITLE Lnet/minecraft/class_2561;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen.mapping]:
```suggestion
	FIELD field_26535 BUFFET_BIOME_TEXT Lnet/minecraft/class_2561;
```

File: mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping [hunk 2]
```diff
@@ -4,6 +4,12 @@ CLASS net/minecraft/class_525 net/minecraft/client/gui/screen/world/CreateWorldS
 	FIELD field_24288 gameRules Lnet/minecraft/class_1928;
 	FIELD field_24588 moreOptionsDialog Lnet/minecraft/class_5292;
 	FIELD field_25478 dataPacksButton Lnet/minecraft/class_4185;
+	FIELD field_25898 GAME_MODE_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26598 ENTER_SEED_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26599 SEED_INFO_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26600 ENTER_NAME_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26601 RESULT_FLOOR_TEXT Lnet/minecraft/class_2561;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping]:
```suggestion
	FIELD field_26601 RESULT_FOLDER_TEXT Lnet/minecraft/class_2561;
```

File: mappings/net/minecraft/client/realms/gui/screen/RealmsConfigureWorldScreen.mapping [hunk 3]
```diff
@@ -20,6 +20,14 @@ CLASS net/minecraft/class_4388 net/minecraft/client/realms/gui/screen/RealmsConf
 	FIELD field_22689 OFF_ICON Lnet/minecraft/class_2960;
 	FIELD field_22690 EXPIRED_ICON Lnet/minecraft/class_2960;
 	FIELD field_22691 EXPIRES_SOON_ICON Lnet/minecraft/class_2960;
+	FIELD field_26479 WORLDS_TITLE Lnet/minecraft/class_2561;
+	FIELD field_26480 CONFIGURE_REALM_TITLE Lnet/minecraft/class_2561;
+	FIELD field_26481 CURRENT_MINIGAME_TITLE Lnet/minecraft/class_2561;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/realms/gui/screen/RealmsConfigureWorldScreen.mapping]:
```suggestion
	FIELD field_26481 CURRENT_MINIGAME_TEXT Lnet/minecraft/class_2561;
```

File: mappings/net/minecraft/client/realms/gui/screen/RealmsUploadScreen.mapping [hunk 4]
```diff
@@ -20,6 +20,8 @@ CLASS net/minecraft/class_4427 net/minecraft/client/realms/gui/screen/RealmsUplo
 	FIELD field_20194 previousTimeSnapshot Ljava/lang/Long;
 	FIELD field_20195 bytesPersSecond J
 	FIELD field_20196 UPLOAD_LOCK Ljava/util/concurrent/locks/ReentrantLock;
+	FIELD field_20503 DOT_TEXTS [Lnet/minecraft/class_2561;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/realms/gui/screen/RealmsUploadScreen.mapping]:
```suggestion
	FIELD field_20503 dotTexts [Lnet/minecraft/class_2561;
```

Comment (haykam821) [reply mappings/net/minecraft/client/realms/gui/screen/RealmsUploadScreen.mapping]: Fixed in ca637d2ab4e446874f1066f968562d0e9185f398

File: mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping (hunk 2 above)
Comment (haykam821) [reply mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping]: Fixed in b79ddfbfb61242b790ce49b0a387c3d27a8e56f9

**Comment by YanisBft** [comment]
@haykam821 mind update and apply the comments please?

**Review by YanisBft — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**